circle.yml

```diff
@@ -1,16 +1,19 @@
 machine:
+  node:
+    version: 8.4.0
   environment:
     SFDX_AUTOUPDATE_DISABLE: true
     SFDX_USE_GENERIC_UNIX_KEYCHAIN: true
     SFDX_DOMAIN_RETRY: 300
-
   pre:
-    - wget -qO- https://developer.salesforce.com/media/salesforce-cli/sfdx-linux-amd64.tar.xz | tar xJf -
-    - ./sfdx/install
     - curl -L -o google-chrome.deb https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
     - sudo dpkg -i google-chrome.deb
     - sudo sed -i 's|HERE/chrome\"|HERE/chrome\" --disable-setuid-sandbox|g' /opt/google/chrome/google-chrome
     - rm google-chrome.deb
+    
+dependencies:
+  post:
+    - npm install -g sfdx-cli
```

Comment:
This needs to go into dependencies so that it happens after node 8.4.0 is installed.